blocks/embed/embed.css

```diff
@@ -4,7 +4,11 @@ main .embed {
   margin: 2rem auto;
   min-height: 12rem;
 }
-  
+
+main .embed.share {
```

Comment:
Is overriding `min-height` the only reason to add this variant? If yes, maybe we can do this by default for all social plugins without the author having to specify in word doc. I am assuming none of the social share buttons require that min-height.

Reply:
At some places plugin require min height like below
https://main--sunstar--hlxsites.hlx.page/_drafts/piyush/how-to-develop-a-healthy-habit